jsons/Policies.json

```diff
@@ -449,6 +452,8 @@
                 "uniques": [
                     "[+2 Production, +2 Food] from every [Land Trade Route (Gold)]",
                     "[+2 Production, +2 Food] from every [Sea Trade Route (Gold)]",
+                    "Gifts of Gold to City-States generate [10]% more Influence",
+                    //cant add "+1 Influence per turn from trade routes to City States, buff gift then"
```

Comment:
Similar discussion here, would prefer to avoid extra buffs for stuff like this. Although... I'm pretty sure extra benefit on Consulates already does something like this

Reply:
ok, removed it